src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/CorporateDto.java

```diff
@@ -0,0 +1,313 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dto.trust;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CorporateDto {
+    @JsonProperty("type")
+    private String type;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("date_became_interested_person_day")
```

Comment:
We've defined dates coming into the API from the web layer as `LocalDate` rather than split fields - we then know that the API can only accept a date. Could this maybe be done here also or are you maybe constrained by the trust spreadsheet and generated JSON?